app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,38 @@
+class ReviewsController < ApplicationController
+  before_action :set_review, only: %i[edit update destroy]
+  
+  def create
+    review = current_user.reviews.build(review_params)
+    if review.save
+      redirect_to toilet_place_path(review.toilet_place.id), notice: 'レビューを投稿しました'
```

Comment:
ToiletPlaces::ApplicationController があればこう書ける

```suggestion
      redirect_to toilet_place_path(@toilet_place), notice: 'レビューを投稿しました'
```